style/forms/ToggleButton.scss

```diff
@@ -0,0 +1,43 @@
+.ds-form-control-toggle {
+  position: relative;
+  align-items: center;
+  display: inline-flex;
+
+  input {
+    display: none;
+  }
+
+  &__slider {
+    position: relative;
+    overflow: hidden;
+    // float: left;
+    border-radius: 37px;
+    cursor: pointer;
+    width: 26.4px;
+    height: 16px;
+    background-color: #d7d7de;
```

Comment:
do we have this color in ScSS variables?

Reply:
I couldn't find it